sql/core/src/main/java/org/apache/parquet/io/ColumnIOUtil.java

```diff
@@ -1,40 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.parquet.io;
-
-/**
- * This is a workaround since methods below are not public in {@link ColumnIO}.
- *
- * TODO(SPARK-36511): we should remove this once PARQUET-2050 and PARQUET-2083 are released with
```

Comment:
This is not a minor. This should be removed with JIRA issue, SPARK-36511 .

Reply:
To @panbingkun , please respect the code when you remove.
Apache Spark community uses IDed TODO like this for trace-ability and we had better respect them.